common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -586,310 +586,6 @@ def test_page_viewed_event(self):
             self.topics_page.visit()
 
 
-@attr(shard=5)
-@ddt.ddt
-class BrowseTeamsWithinTopicTest(TeamsTabBase):
```

Comment:
Do we have any ideas on why the setUp failed?  I think it would help to understand this better before we delete the whole set of tests here.

Reply:
Someone else just saw the JIra ticket I made and commented that the issue is with bokchoy surver rather than the test being flacky. I'll look further into this, but, most likely, will remove this change soon.

Reply:
link to jira ticket: [here](https://openedx.atlassian.net/servicedesk/customer/portal/9/CR-926?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJxc2giOiI5NTUzNTJkZmM0NjJiYTFjZGVhODk0NWY1ZjgxZjhkYmM4NTk0MTZkZDlkNzM3YmIwYzA0MGFiNDBhYTU3MDA3IiwiaXNzIjoic2VydmljZWRlc2stand0LXRva2VuLWlzc3VlciIsImNvbnRleHQiOnsidXNlciI6IjQzMjI0IiwiaXNzdWUiOiJDUi05MjYifSwiZXhwIjoxNTYzMzQyNDUwLCJpYXQiOjE1NjA5MjMyNTB9.kxhoDlI7eqhZG38c2vzH3gL90dOq1RVbBV8163sdQq4)